streams/src/test/java/org/apache/kafka/streams/tests/EosTestClient.java

```diff
@@ -112,8 +102,8 @@ private KafkaStreams createKafkaStreams(final Properties props) {
         props.put(StreamsConfig.NUM_STREAM_THREADS_CONFIG, 1);
         props.put(StreamsConfig.NUM_STANDBY_REPLICAS_CONFIG, 2);
         props.put(StreamsConfig.REPLICATION_FACTOR_CONFIG, 3);
-        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, StreamsConfig.EXACTLY_ONCE);
         props.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0);
+        props.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 5000); // increase commit interval to make sure a client is killed having an open transaction
```

Comment:
Small side improvement.

Reply:
Could you elaborate a bit? The comment doesn't seem readable.

Reply:
Comment says:
```
// increase commit interval to make sure a client is killed having an open transaction
```

If we commit with small commit interval, the probability that there is no pending transaction when we kill the instance is high.

We delay to start a new transaction until we do the first `send()` and would commit quickly afterwards. If we "stall" in between waiting for new data (what is not uncommon in this test) there will be no open tx for some time.

During debugging I did some segment dumps and could not find a single aborted transaction.

Does this make sense?

Reply:
kk, I think I get the motivation, but still feels `a client is killed` and `having an open transaction` could not be connected as a full sentence here. Maybe we should call `a killed client`?